04_run_batch_inference.ipynb

```diff
@@ -618,11 +655,11 @@
    "outputs": [],
    "source": [
     "@remote(\n",
-    "    instance_type=\"ml.g6e.xlarge\",  # Powerful GPU for fast inference\n",
+    "    instance_type=instance_type,  # Powerful GPU for fast inference\n",
     "    instance_count=1,  # Single instance for cost efficiency\n",
     "    volume_size=200, # Large volume for model and data storage\n",
     "    job_name_prefix=job_name_prefix,\n",
-    "    # use_spot_instances=True, # Cost efficient inference. Inference can be restarted if no spot capacity. \n",
+    "    # use_spot_instances=True, # Enable cost efficient inference. Inference can be restarted if no spot capacity. \n",
```

Comment:
It's not a problem, but I wonder if there was a particular reason we have spot enabled by default in training but disabled by default in inference?